compiler/src/dotty/tools/dotc/config/CompilerCommand.scala

```diff
@@ -71,7 +71,14 @@ object CompilerCommand extends DotClass {
       val ss                  = (ctx.settings.allSettings filter cond).toList sortBy (_.name)
       val width               = (ss map (_.name.length)).max
       def format(s: String)   = ("%-" + width + "s") format s
-      def helpStr(s: Setting[_]) = s"${format(s.name)} ${s.description}"
+      def formatSettings(name: String, value: String) = if (value.nonEmpty) format("\n") ++ s"  $name: $value." else ""
```

Comment:
Hm, why do we need to use `format` on `\n`? I'm probably being slow (or rushed)... 
And expressions can go inside the s formatter...
Any chance we can use `s"\n${format(name)} ${format(value)}"`? That's closer to what we do in `helpStr`, and hopefully clearer.

Reply:
I use `format("\n")` as a hack to start the second line with an empty first column for Default or Choices
<img width="716" alt="screen shot 2018-02-20 at 16 12 08" src="https://user-images.githubusercontent.com/6079280/36431959-44c715ee-1659-11e8-829e-94ff1899afc6.png">